src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ProjectUpgradesPage/versions/versions.ts

```diff
@@ -0,0 +1,26 @@
+import { t } from '@lingui/macro'
+import { UpgradeController } from './upgradeForms/JBController3_0_1'
+import { UpgradeFundingCycle } from './upgradeForms/V3/UpgradeFundingCycle'
+
+export type JBVersion =
+  | '3' // upgrades to layer 2 contracts
+  | 'JBController3_0_1' // upgrades to a new JBController
+
+export const VERSIONS: {
```

Comment:
this seems less like "versions" and more "types of upgrades"

feels more intuitive to me to have a `type UpgradeOption = 'v3_funding_cycles' | 'v3_controller'` instead of `type JBVersion = '3' | 'JBController3_0_1 '`

am i thinking about this the right way?

Reply:
yep great great call out. totally agree, will change to something like `UpgradeOption` 👍